advocacy_docs/edb-postgres-ai/analytics/reference/queries.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Queries
+description: Supported queries in Lakehouse and best practices when composing them
+---
+
+In general, **READ ONLY** queries are supported. You cannot write directly to
+object storage. This includes all Postgres built-in functions, statements
+and types. It also includes any of those provided by EPAS or PGE, depending on
+which distribution you choose to deploy.
+
+In general, you cannot insert, update, delete or otherwise modify data. You
+cannot `CREATE TABLE`. You must load data into the bucket out-of-band, either
+with your own ETL scripts or with Lakehouse Sync.  See "Advanced: Bring Your Own
+Data" for more details. (In the future, we will be making this more usable with
+a custom DDL).
+
+One exception is Postgres system tables, such as those used for storing users,
+roles, and grants. These tables are stored on the local block device, which is
+included in backups and restores. So you can `CREATE USER` or `CREATE ROLE` or
+`GRANT USAGE`, and these users/roles/grants will survive restarts and restores.
+
+## Gotcha: Do not set `search_path`
+
+Do not set `search_path`. Always reference fully qualified table names.
```

Comment:
@mildbyte Is this warning still true? (do not set `search_path`)

Reply:
No, we now support search_path, you can delete that gotcha entirely